lib/repl.js

```diff
@@ -320,9 +323,21 @@ function REPLServer(prompt,
           // value for statements and declarations that don't return a value.
           code = `'use strict'; void 0;\n${code}`;
         }
+        let pwd;
+        try {
+          const { pathToFileURL } = require('url');
+          pwd = pathToFileURL(process.cwd()).href;
+        } catch {
+        }
+        const asyncESM = require('internal/process/esm_loader');
```

Comment:
Should that not be loaded at the top level?

Reply:
could be, but i don't think we eagerly load esm_loader at top in most places since it is experimental/flagged, we do so as late as possible like in https://github.com/nodejs/node/blob/85898e0aca26cee2e68aea9ba384e23cd4457849/lib/internal/bootstrap/pre_execution.js#L388 . we could probably make this even lazier though now that i think about it.

Reply:
moved up but remaining in `defaultEval`; made it lazier.